pkg/kubelet/dockertools/manager.go

```diff
@@ -308,6 +308,28 @@ type containerStatusResult struct {
 	err    error
 }
 
+// determineContainerIP determines the IP address of the given container.  It is expected
+// that the container passed is the infrastructure container of a pod and the responsibility
+// of the caller to ensure that the correct container is passed.
+func (dm *DockerManager) determineContainerIP(podNamespace, podName string, container *docker.Container) string {
```

Comment:
this function was deleted last week. I re-introduced this function so that the ip can be used in /etc/hosts file